api/v1/dois/PKPDoiHandler.php

```diff
@@ -181,7 +181,7 @@ public function get(SlimRequest $slimRequest, APIResponse $response, array $args
 
         // The contextId should always point to the requested contextId
         if ($doi->getData('contextId') !== $this->getRequest()->getContext()->getId()) {
-            return $response->withStatus(403)->withJsonError('api.dois.400.contextsNotMatched');
+            return $response->withStatus(403)->withJsonError('api.dois.403.contextsNotMatched');
```

Comment:
The locale key, `api.dois.403.contextsNotMatched`, does not exist in OMP or OPS. Does it need to be added?